src/core/nonbonded_interactions/nonbonded_interaction_data.hpp

```diff
@@ -36,6 +36,201 @@ constexpr double INACTIVE_CUTOFF = -1.;
 /* Data Types */
 /************************************************************/
 
+/** \name Lennard-Jones with shift */
+/*@{*/
+struct LJ_Parameters {
```

Comment:
In doxygen, groups of 1 element are not meaningful, and any struct/class defined in them are automatically taken out, leaving the group empty and the struct without a description. In this .hpp file, all non-bonded IAs of the form
```c++
/** \name Lennard-Jones with shift */
/*@{*/
struct Something {
...
};
/*@}*/
```
should become
```c++
/** Lennard-Jones with shift */
struct Something {
...
};
```